lifecycle-operator/config/samples/lifecycle_v1beta1_keptnappversion.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: lifecycle.keptn.sh/v1beta1
+kind: KeptnAppVersion
+metadata:
+  name: keptnappversion-sample
+status:
```

Comment:
There is no need to specify status fields, it would be better to specify and show how the spec fields are used